build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
 	compile("org.springframework:spring-context-support")
 	compile("org.springframework:spring-core")
 	compile("org.springframework:spring-expression")
-	compile("com.blackducksoftware.integration:hub-common:18.4.1")
+	compile("com.blackducksoftware.integration:hub-common:+")
```

Comment:
Do we want to leave this as a dynamic version. This may not produce consistent, reproducible build results if we ever have to respin the 2.0.4 build. Also, this may cause local builds in the future to fail or the output of the resulting binary to be inconsistent.

I can see this being useful during development, especially when we are expecting changes in the dependent library, but once it gets to release I would think we would want to specify exactly what version of the dependent library we used.

Reply:
ok Jason. Will modify then.